src/Cli/dotnet/commands/dotnet-sln/LocalizableStrings.resx

```diff
@@ -183,4 +183,10 @@
   <data name="SlnxGenerated" xml:space="preserve">
     <value>.slnx file {0} generated.</value>
   </data>
+  <data name="CannotMigrateSlnx" xml:space="preserve">
+    <value>Cannot migrate .slnx file.</value>
+  </data>
+  <data name="SerializerNotFound" xml:space="preserve">
+    <value>Could not find serializer for file {0}.</value>
+  </data>
```

Comment:
I wouldn't expose the concept of a 'serializer' to end users. Suggest keeping it more high-level, something like:

```
Couldn't read solution file {0}. Supported file types are .sln and .slnx, and must be valid solutions.
```